frontend/src/app/account/account.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="container">
+  <div class="row">
+    <div class="col-6">
+      <div class="card">
+        <h3 class="card-header">Pictures</h3>
```

Comment:
Two things here - it's in English and after all we abandoned (at least for the time being) user profile pictures - see: #993